ivy/functional/frontends/paddle/manipulation.py

```diff
@@ -44,6 +44,34 @@ def cast(x, dtype):
     return ivy.astype(x, dtype)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("bool", "int32", "int64", "float16", "float32", "float64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def chunk(x, chunks, axis=0, name=None):
+    if x.shape == ():
+        if chunks > 1:
```

Comment:
You could join this condition in the line above using `and`

Reply:
resolved thanks @bipinKrishnan